### PR TITLE
Add python3.9

### PR DESCRIPTION
**Pull request by aperezhortal**
Add Python3.9 build.

Checklist
* [x] Used a fork of the feedstock to propose changes ([see here why](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests))
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,20 +19,16 @@ requirements:
...
-    - openmp 4.0.*  # [osx]
-    - jsmin
-    - scipy
-    - matplotlib-base
-    - jsonschema
+    - openmp # [osx]
   run:
     - python
-    - openmp 4.0.*  # [osx]
+    - openmp # [osx]
```
Comment (dnerini) [inline recipe/meta.yaml:28]:
```suggestion
    - openmp  # [osx]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,20 +19,16 @@ requirements:
...
-    - numpy 1.16.*
+    - numpy >=1.17
     - cython
     - pip
-    - openmp 4.0.*  # [osx]
-    - jsmin
-    - scipy
-    - matplotlib-base
-    - jsonschema
+    - openmp # [osx]
```
Comment (dnerini) [inline recipe/meta.yaml:25]:
```suggestion
    - openmp  # [osx]
```

**Comment by dnerini** [comment]
Still failing for 3.9 on win-64...

Not sure if we can fix anything on our side? 
Alternatively, what if we simply skip the failing built so that we can at least publish the builts for Linux and osx?

**Comment by aperezhortal** [comment]
> Alternatively, what if we simply skip the failing built so that we can at least publish the builts for Linux and osx?

I agree. It seems that a few packages need to catch up with python 3.9 on windows. 

I'll try to fix the recipe to ignore  win+python3.9.

**Review by dnerini — APPROVED** [review]

**Pull request merged**